src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -45,6 +45,33 @@ export default class NavigationLibrary {
     }
 
     /**
+     *
+     * @property drawSids
+     * @return
+     */
+    get drawSids() {
```

Comment:
I'm pretty certain this doesn't actually draw the SIDs, right? Looks to me like it just retrieves the lines needed to be drawn, and then when this method is called, the caller is the one doing the canvas work. Perhaps renaming to `get sidLines` or something if that is indeed the case?